ci/ci/constants.py

```diff
@@ -26,6 +26,7 @@ def __init__(self, gh_username: str, hail_username: Optional[str] = None, teams:
     User('chrisvittal', 'cvittal', HAIL_TEAM),
     User('cjllanwarne', 'chrisl', HAIL_TEAM),
     User('ehigham', 'ehigham', HAIL_TEAM),
+    User('grohli', 'grohlice', SERVICES_TEAM),
```

Comment:
If you click through to the CI error message it looks like - 
```
/io/repo/ci/ci/constants.py
  /io/repo/ci/ci/constants.py:29:32 - error: Argument of type "Literal['Services']" cannot be assigned to parameter "teams" of type "List[str] | None" in function "__init__"
    Type "Literal['Services']" is not assignable to type "List[str] | None"
      "Literal['Services']" is not assignable to "List[str]"
      "Literal['Services']" is not assignable to "None" (reportArgumentType)
1 error, 0 warnings, 0 informations 
```

I think the type mismatch is because `HAIL_TEAM`is a list whereas `SERVICES_TEAM` is a string 🤦‍♂️ . Presumably all we need to do is add some `[]`s